woocommerce/payment-gateway/admin/class-sv-wc-payment-gateway-admin-order.php

```diff
@@ -137,9 +137,11 @@ protected function enqueue_edit_order_assets( \WC_Order $order ) {
 	 * @since 5.0.0
 	 */
 	public function maybe_add_capture_charge_bulk_order_action() {
-		global $post_type, $post_status;
+		
+		global $post_type, $post_status, $current_screen;
 
-		if ( ! current_user_can( 'edit_shop_orders' ) ) {
+
+		if ( ! current_user_can( 'edit_shop_orders' )  && $current_screen->id === SV_WC_Order_Compatibility::get_screen_id()) {
 			return;
 		}
 
```

Comment:
`$post_type` and `$post_status` will not work in WooCommerce to check post type and post status. We need more helper methods for these operations.  I'm working on it.